message_sender/serializers.py

```diff
@@ -21,9 +24,7 @@ class Meta:
 
     def to_internal_value(self, data):
         if data.get('channel'):
-            channel_id = data.pop('channel')
-            data['channel_channel_id'] = channel_id
-
+            channel_id = data.get('channel')
             if not Channel.objects.filter(channel_id=channel_id).exists():
                 raise IncorrectChannel()
```

Comment:
Do we still need this check? From my understanding of the PrimaryKeyRelatedField, `data['channel']` should be a `Channel` object, not a string anymore.